Specs/DateTime/Hindi/SetParser.json

```diff
@@ -172,6 +172,7 @@
     "Context": {
       "ReferenceDateTime": "2017-09-27T12:25:54.2909444+03:00"
     },
+    "Comment": "The input literally translates to 'I'll leave each in two days' and 'in two days' is processed as Date instead of Duration",
```

Comment:
Is this because the translation was incorrect from the corresponding case in English? If so, can we add a new case with the correct translation?
This one can be moved to DateParser then (if not duplicate).

Reply:
The problem is the translation is correct but the structure of the pattern is different in Hindi, and attempts to support it break other sub-entities. The solution seems to require Hindi specific modifications of the base code, so we have skipped it for now.

Reply:
Got it. I misunderstood the comment at first.